02-membership edits: rights, obligations, self-curation rationale (#172)
protocolguild/documentation: docs/02-membership.md

The final step of the formula uses a Square Root to compress the weight range. This is done to not overly privilege long-term members over newer contributors.

> #### Time-Weight Example

Reviewer: ```suggestion
#### Time Weight Example
```

![RelativeWeights](https://github.com/cheeky-gorilla/membership/assets/76262359/93a6f5a0-393e-46c8-8dfd-a51aa6864f46)

### Governance

> Only members are invited to govern the mechanism. This may include:

Reviewer 2: The phrasing here feels too "outside looking", I'd reframe as "Protocol Guild members govern the evolution of the mechanism, which includes:"

I'd also add a note about how governance does **not** have control over any vesting funds. PG members can't vote to "cancel the vest and get all the tokens now".

- modifying high-level project elibility
- discussing which smart contracts to use, and how to operate them
- setting parameters like how long vests should be
- how and where to fundraise

Typically this takes the shape of rough consensus discussions. In some special cases, decisions needs to be voted on or ratified - learn more in smart contract section.

## 2.4 Obligations

Members are expected to participate in curation. Consider these examples:

- an individual updating their personal status
- a colleague making updates on behalf of their team
- members giving input to shape the addition/removal process

### Self-Curation

> We use the term "self-curation" to describe how the membership curates its own beneficiary set. This is an important part of the funding mechanism, and what distinguishes it from others e.g. badgeholders in OP Retro, or Gitcoin's use of donors and quadratic weighting. Curatorial bodies external to the beneficiary set come with their own tradeoffs. Self-curation is suited for this context for a few reasons:

Reviewer 2: Avoid the circular definition, and consider removing explicit references to other mechanisms. 

```suggestion
We use the term "self-curation" to describe how the membership selects its own beneficiary set. This is an important distinction between Protocol Guild and other public goods funding mechanisms. While curatorial bodies external to the beneficiary set are appropriate in some contexts, self-curation is well suited here for a few reasons:
```

> 1. Local actors are the only ones with domain knowledge

Reviewer 2: ```suggestion
1. Local actors have the most domain knowledge
```

Others have knowledge too, just less of it 😄

Discussion should be open for at least one week to give members time to review and discuss.

Bias or conflicts of interest of the nominator should be disclosed, if they exist, e.g. where one is an advisor to the other’s side project. After one week of rough consensus, the PR should be merged or rejected.

### Removing a Member

#### Self-Removal, Affiliation, Weights

Removals from the active membership should come from the member themselves, i.e. self-removal, by opening a PR in the [same repo](https://github.com/protocolguild/documentation). Example [here](https://github.com/protocolguild/documentation/pull/152). Where this isn't possible due to extenuating circumstances, the member should be notified or tagged on the PR so they are aware of the changes.

Affiliation and weight changes should include some rationale for the change, ideally from the member themselves and seconding by a colleague.

#### Peer Removal

Peer removals can occur when a member stops contributing to eligible work, and is not responsive to requests to self-remove (as described above). In such cases, another member (ideally from the same team) can propose to remove the inactive member.

> There may also be situations when another member proposes the removal of an existing member, even if they are continuing with eligible work. This should only happen in special circumstances where the cost of contention is higher than the loss of institutional legitimacy which their continued membership would produce. The PR should include ample references to the alleged misconduct and justification for the removal. To date, this method has never been used.

Reviewer: ```suggestion
There may also be situations when another member proposes the removal of an existing member, even if they are continuing with eligible work. This should only happen in special circumstances where the cost of contention is lower than the loss of institutional legitimacy which their continued membership would produce. The PR should include ample references to the alleged misconduct and justification for the removal. To date, this method has never been used.
```

> There may also be situations when another member proposes the removal of an existing member, even if they are continuing with eligible work. This should only happen in special circumstances where the cost of contention is higher than the loss of institutional legitimacy which their continued membership would produce. The PR should include ample references to the alleged misconduct and justification for the removal. To date, this method has never been used.

Reviewer 3: ```suggestion
There may also be situations when another member proposes the removal of an existing member, even if they are continuing with eligible work. This should only happen in special circumstances where the cost of contention is higher than the loss of institutional legitimacy which their continued membership would produce. The PR should include ample documentation and justification for the removal. To date, this method has never been used.
```

I think removing the "misconduct" here allows this to be more generally applicable since I believe it's reasonable to expect there to be reasons beyond "misconduct" that might arise.

Reviewer 2: @macfarla I think you can make a case that "higher" is also correct, and @pipermerriam I agree with your point re: broadening. My takeaway after re-reading the paragraph 3 times is we should just remove that sentence to simplify things. 

```suggestion
There may also be situations when another member proposes the removal of an existing member, even if they are continuing with eligible work. This should only happen in special circumstances where the removed member's contributions are far below what would generally be expected, or in the case of other grave misconducts. The PR should include ample documentation and justification for the removal. To date, this method has never been used.
```

Author: yeah, the wording could be improved

> ## 2.6 Quotes

Reviewer: ```suggestion
## 2.5 Quotes
```
